src/Tickets/Emails/Preview_Data.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Tickets Emails Preview Data class.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
```

Comment:
Given that we have the `Preview_Modal` in the `Admin` folder, would it also make more sense to have the `Preview_Data` there?

Reply:
Done